### DISCUSSION TITLE
Creating a user via GraphQL Results in "Mail Template Failed to Load" Error

### DISCUSSION THREAD

**Discussion by willricketts** [requarks/wiki#5683] [Error / Bug Report]
Greetings,

I'm running into an issue while creating non-administrative member accounts via GraphQL api. When I run the mutation to create a user, the user gets created with the correct group membership, however if I set `sendWelcomeEmail` to true in the query, I'm returned the following error (elixir):

```elixir
"responseResult" => %{
  "message" => "Mail template failed to load.",
  "slug" => "MailTemplateFailed",
  "succeeded" => false
},
```

The following is the mutation query I'm using:

```graphql
mutation {
  users {
    create (
      email: "foo@bar.com"
      name: "Foo Bar"
      passwordRaw: "foobar123"
      providerKey: "local"
      groups: [3]
      mustChangePassword: true
      sendWelcomeEmail: true
    ) {
        responseResult {
          succeeded
          slug
          message
      }
      user {
        id
      }
    }
  }
}

```

Again, when I hop over to admin user management, my new user has been created, though the response from the API indicates the contrary and the welcome email isn't sent.

Any ideas?

**Comment by bartmkamphuis** [comment]
In https://github.com/requarks/wiki/discussions/5494 it is stated that the Welcome email is not implemented yet.
Work around this by setting sendWelcomeEmail: false will result in 
```
{
  "data": {
    "users": {
      "create": {
        "responseResult": {
          "succeeded": true,
          "slug": "ok",
          "message": "User created successfully"
        },
        "user": null
      }
    }
  }
}
```

**Reply by willricketts** [reply]
Gotcha. Thanks for the response. Is the welcome email feature one that's coming soon? I ask because it's critical to my use case! Thanks :)
